lib/fog/libvirt/models/compute/server.rb

```diff
@@ -90,9 +90,9 @@ def disk_path
           volumes.first.path if volumes and volumes.first
         end
 
-        def destroy(options={ :destroy_volumes => false})
+        def destroy(options={ :destroy_volumes => false, :flags => 0 })
           poweroff unless stopped?
-          service.vm_action(uuid, :undefine)
+          service.vm_action(uuid, :undefine, options[:flags])
```

Comment:
Can you pass the flags only if they are non-null? That would be backward compatible with any version. Not that this would be a huge problem, 7 years is enough in IT but...

Sorry for the delay.

Reply:
My remark still stands, please just pass flags only if they are present. I cannot guarantee that passing flags: 0 will work against hypervisor running on older RHEL7.

Reply:
Sorry, I completely forget about this PR.